Packs/CommonScripts/ReleaseNotes/1_7_7.md

```diff
@@ -0,0 +1,5 @@
+
+#### Scripts
+##### HttpV2
+- Updated the Docker image to: *demisto/python3:3.10.4.30607*.
+- Added support for passing the *headers* parameter as string.
```

Comment:
Please make sure to update the `yml` description and update the readme accordingly.